cmd/e2equeries/main.go

```diff
@@ -63,7 +65,7 @@ func main() {
 	}
 	if !foundRekey {
 		// this will report the error in a handy way
-		printAccountQuery(db, idb.AccountQueryOptions{EqualToAuthAddr: rekeyedAuthAddrs[0][:], Limit: 1})
+		printAccountQuery(db, idb.AccountQueryOptions{EqualToAuthAddr: rekeyedAuthAddrs[0][:], Limit: 0})
```

Comment:
What error is this supposed to be reporting?

Reply:
I think I see it. This triggers the "came up short" error?

Reply:
yes